releases/point-2023-2-release-note.md

```diff
@@ -0,0 +1,86 @@
+---
+description: This article lists new features, improvements, and bug fixes in SysKit Point version 2023.2.
+--- 
+
+# SysKit Point 2023.2
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think.](https://www.syskit.com/company/contact-us/)
+
+**Product version:** 2023.2.0
+
+**Build number:** xxx
+
+**Release date:** May 11, 2023
+
+## Get the Latest Version
+
+### SysKit Point
+
+* **New to SysKit Point?** Get the latest SysKit Point version from [Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/syskitltd.syskit_point).<br/>
+   Find all steps described in detail in the [deployment article](../set-up-point-data-center/deployment/deploy-syskit-point.md).
+    
+* **Already using SysKit Point?** Upgrade to the latest SysKit Point version. <br/>
+[![](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fsyskitassetsstorage.blob.core.windows.net%2Fpoint%2FARMTemplates%2FPointUpdateDeploy%2FPointUpdateTemplate.json)
+
+See detailed step-by-step instructions on how to upgrade SysKit Point in the [upgrade article](../set-up-point-data-center/deployment/upgrade-syskit-point.md).
+
+## Features
+
+* **You can now resolve access and provisioning requests in the SysKit Point web app**! Whether using the SysKit Point Teams app or the SysKit Point web app, you can now resolve these tasks in any of the two places. 
+As before, you will:
+    * **Get a notification in the Teams app**
+    * **Receive an email that opens the request in SysKit Point Teams app** 
+    * **Be able to resolve access and provisioning requests within SysKit Point Teams app**
+In the newest addition, you will:
+    * **Get a task in the SysKit Point app as well**
```

Comment:
In line 29 we are mentioning the SysKit Point **web** app as opposed to the Teams, so maybe we should keep using **teams** and **web** across the article?

Reply:
Added 'web' in line 35. If you see other places where this could be applied, let me know. 
I'm fine with the section describing tasks in Point web app having the web vs teams app emphasis. Further RN article doesn't need it that much I think.